scripts/publish-instance-metrics/on-start.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+# OVERVIEW
+# This script publishes the system-level metrics from the Notebook instance to Cloudwatch.
+#
+# Note that this script will fail if either condition is not met
+#   1. Ensure the Notebook Instance has internet connectivity to fetch the example config
+#   2. Ensure the Notebook Instance execution role permissions to cloudwatch:PutMetricData to publish the system-level metrics
+#
+# https://aws.amazon.com/cloudwatch/pricing/
+
+# PARAMETERS
+NOTEBOOK_INSTANCE_NAME=MyNotebookInstance
```

Comment:
To self contain this,  you could fetch the Notebook Instance name as per https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-metadata.html

Reply:
Looks good apart from this.

Reply:
Oh! That is perfect. I was about to request a PFR to allow users to pass command line arguments to the LC script. 
This makes more sense as we can have other arguments as tags to the notebook instance.